frame/evm/src/tests.rs

```diff
@@ -354,3 +354,72 @@ fn handle_sufficient_reference() {
 		assert_eq!(account_2.sufficients, 0);
 	});
 }
+
+#[test]
+fn test_hotfix_inc_account_sufficients_requires_signed_origin() {
+	new_test_ext().execute_with(|| {
+		let addr = H160::from_str("1230000000000000000000000000000000000001").unwrap();
+		<crate::AccountCodes<Test>>::insert(addr, &vec![0]);
+
+		let result = EVM::hotfix_inc_account_sufficients(Origin::root(), vec![addr]);
```

Comment:
Is there a better way here to explicitly have an unsigned `Origin`?

Reply:
Good question, maybe `Origin::none()`?

Reply:
I'm believe `Origin::none()` signifies a _lack_ of origin as defined by the enum here:

```rust
pub enum RawOrigin<AccountId> {
    Root,
    Signed(AccountId),
    None,
}
```

> None: A lack of origin. This needs to be agreed upon by the validators or validated by a module to be included. This origin type is more complex by nature, in that it is designed to bypasses certain runtime mechanisms. One example use case of this origin type would be to allow validators to insert data directly into a block.

Reply:
I see, then I don't know. I agree `Origin::root` is not very _self-explanatory_ for this use case.